lib/hawkular/inventory/inventory_api.rb

```diff
@@ -429,6 +429,17 @@ def create_metric_type(feed_id, metric_type_id, type = 'GAUGE', unit = 'NONE', c
       MetricType.new(new_mt)
     end
 
+    # List operation types for given resource type
+    # @param [String] resource_type_path canonical path of the resource type entity
+    # @return [Array<String>] List of operation type ids
+    def list_operation_types(resource_type_path)
```

Comment:
Hmm, I can add that, but rather than overloading the method and doing some is_a? String ruby-fu, I'd create a new method called `list_operation_definitions_for_resource(resource_path)`, wdyt?